app/cmd.go

```diff
@@ -524,6 +535,12 @@ func getConfig() (appConfig, error) {
 
 func init() {
 	RootCmd.Flags().StringP("listen", "l", "localhost:8081", "Listen address:port for the application.")
+	RootCmd.Flags().StringP("listen-tls", "t", "", "HTTPS listen address:port for the application.  Requires setting --tls-cert and --tls-key OR --tls-cert-file and --tls-key-file.")
+
+	RootCmd.Flags().String("tls-cert-file", "", "If using listen-tls, specify path of PEM-encoded certificate file.")
```

Comment:
nit: use descriptive phrasing instead of instructional.

Something like "Specifies a path to a PEM-encoded certificate. Has no effect if `--listen-tls` is unset.)"